docs/en_US/network-dumps.md

```diff
@@ -0,0 +1,69 @@
+
+### All (WireShark)
+If neither solution above works, the only option you have is to capture the traffic manually off the console. This is easiest done with WireShark and a hosted wifi access point. Create a hosted wifi access point on your PC which has WireShark installed and select that access points interface in WireShark. Connect the console to this hosted access point, and you should begin to see all the network traffic from your console in WireShark. In order to both create a hosted access point and keep your PC online, you must have 2 ways of connecting to the internet on the PC. This is easiest done by using a USB wifi adapter to host the access point, and using ethernet to remain online. Not all USB wifi adapters support hosting access points. Ensure the one you use does. [This adapter](https://www.amazon.com/dp/B07C9TYDR4) is relatively cheap and supports hosting access points.
+
+Just like with HokakuCafe, traffic dumped with WireShark will be encrypted. In order to decrypt the traffic, your NEX account username and password must also be submitted. Your NEX account has no relation to any other account, such as your NNID, and cannot be used to gain any information about you. It can, however, allow someone to log into game servers as you. Ensure you only provide these details to trusted individuals.
+
+- 3DS - To dump your NEX username and password from a 3DS download and run [this homebrew application](https://9net.org/~stary/get_3ds_pid_password.3dsx). This will create a file on your SD card named `nex-keys.txt` with your NEX username and password. Copy and paste the contents of this file into the message of your submission.
+- Wii U - To dump your NEX username and password from a Wii U connect to your Wii U with an FTP client. Navigate to `/storage_mlc/usr/save/system/act` and download all the folders inside this folder. Check the `account.dat` file in each folder and look for your NNID username in the `AccountId` field. Once found, in the same `account.dat` file locate both the `PrincipalId` and `NfsPassword` fields. Copy their values and paste them into the message of your submission.
+
+# HTTP Packets
+Some games use HTTP requests for some features. Additionally, non-game titles will often use HTTP requests for their services. The above methods will not always capture HTTP requests in a way that is usable, if at all. SpotPass data is also downloaded via HTTP requests. For this, an HTTP proxy server is required. There are several options for HTTP proxy servers, which all depend on your operating system, skill level, and, for one, your budget
+
+TODO - We have a NoSSL patch for the Wii U, but no builds are uploaded anywhere. Where should we upload this? Discord won't work anymore because of the expiring download link update. 3DS NoSSL patch by Billy https://github.com/SciresM/3DS-SSL-Patch/pull/2
```

Comment:
We could use a GitHub raw link to the repo: https://github.com/InternalLoss/3DS-SSL-Patch/raw/master/code.ips

A common comment should be added about requiring dumps of the console certificates

Reply:
We don't have a public repo for the Wii U NoSSL patch. It's always just been an internal tool, it's in a private repo here https://github.com/PretendoNetwork/wiiu-nossl. Can downloads still be done from private repos if users don't have access to the repo?

Reply:
oh my bad, I didn't notice you are talking about the Wii U patch. You can't download from private repos, but what are your thoughts on opening that repo to the public? I don't think it needs to be hidden if we are going to distribute the patch anyway

Reply:
I asked Quarky earlier today about why it was private. Waiting on a reply back before opening it back up. For now both SSL patches are included in the public mitmproxy repo

Reply:
The answer is because it's a development tool and may confuse regular users who just install everything from the org